scripts/lint-sh.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
```

Comment:
Missing `set -e`, so only errors in the files covered by the last `find` command will actually make the script return an error status.